platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/EventWindow.java

```diff
@@ -28,10 +28,9 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 /**
- * Determines the non-ancient lower bound (inclusive) on events and communicates the window of rounds between the
- * pendingConsensusRound and the minimumRoundNonAncient (inclusive).
+ * Describes the current window of events that the platform is using.
  */
-public class NonAncientEventWindow {
+public class EventWindow {
```

Comment:
Searching for regex `non.?ancient.?event.?window` yields a couple missed instances of the old terminology

Reply:
I only found one (although there may have been a few I fixed based on your previous comments). I fixed the one that I did find.